src/cephadm/cephadm.py

```diff
@@ -5018,7 +5018,7 @@ def parse_mon_addrv(addrv_arg: str) -> List[EndPoint]:
     addrv_args = []
     addr_arg = addrv_arg
     if addr_arg[0] != '[' or addr_arg[-1] != ']':
-        raise Error(f'--mon-addrv value {addr_arg} must use square backets')
+        raise Error(f'--mon-addrv value {addr_arg} must use square brackets')
```

Comment:
This fixed the spelling here, but we also need to fix it in the corresponding unit test to get the test passing (that's the reason make check is failing). Here's a sample commit changing the test to get it passing https://github.com/adk3798/ceph/commit/abd9b27454c6b12eb131384379adee710504396e

Reply:
So, the reason it wasn't changed is that the file ended up in excludes:
https://github.com/ceph/ceph/blob/32fcb4a03159d25547dd7f87e5d7176e1a0e5ed3/.github/actions/spelling/excludes.txt#L155

I'm not sure offhand _why_ it ended up there. I could review the runs in https://github.com/check-spelling/ceph/actions/workflows/spelling.yml to try to figure out why. But, I'd rather not spend the time doing that unless we think it indicates a bug in the analysis/detection of things to skip, and that should turn up pretty quickly when I remove the excludes entry.